test/crypten/test_context.py

```diff
@@ -1,8 +1,32 @@
 import pytest
+import crypten
+
+import torch as th
 import syft as sy
+
 from syft.frameworks.crypten.context import run_multiworkers
-import torch as th
-import crypten
+from syft.frameworks.crypten.worker_support import methods_to_add
+
+
+def test_add_crypten_support(workers):
+    alice = workers["alice"]
+
+    for method in methods_to_add:
+        assert not hasattr(
+            alice, method.__name__
+        ), f"Worker should not have method {method.__name__}"
+
+    alice.add_crypten_support()
```

Comment:
What exactly is involved in 'Adding crypten support?" Why would it not be supported by default?

Reply:
There was a discussion [here] (https://github.com/OpenMined/PySyft/pull/3254).
Also, added screenshots - I hope I did not miss any information
(http://swarm.cs.pub.ro/~gmuraru/OpenMined/pic1.png)
(http://swarm.cs.pub.ro/~gmuraru/OpenMined/pic2.png)

**TL;DR:** We discussed that maybe the framework information might not be contained in the ```BaseWorker``` and we should have a mechanism to register those "specific framework methods". If this is not wanted, I can switch to something else :D